velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -2847,3 +2848,22 @@ TEST_F(DateTimeFunctionsTest, dateParse) {
   VELOX_ASSERT_THROW(
       dateParse("116", "%y+"), "Invalid format: \"116\" is malformed at \"6\"");
 }
+
+TEST_F(DateTimeFunctionsTest, currentDate) {
+  auto tz = "America/Los_Angeles";
+  auto mockRowVector =
+      makeRowVector({BaseVector::createNullConstant(UNKNOWN(), 1, pool())});
+  setQueryTimeZone(tz);
```

Comment:
Perhaps, add a test when tz is not set, e.g. verify code path where timeZone_ is nullptr.